### ISSUE TITLE
Windows Installation Failure: Not Enough Disk Space

### ISSUE THREAD

**Issue by joesixpack** [metahashorg/metagate#15]
![image](https://user-images.githubusercontent.com/23145642/61464026-6b929580-a92a-11e9-9964-8466e8798086.png)

**Comment by maxwelfox** [comment]
Hi! Did you manage to solve the problem ?

**Comment by magisterRab** [comment]
Which is the installation directory?

**Comment by joesixpack** [comment]
No, the problem still exists on Windows v1.19.7.  Installation is either to user dir on C or to a different drive.  Cleaning temp dirs did not help.

**Comment by joesixpack** [comment]
I can confirm this works okay on Windows 10 just not 7 when installed to a different drive.
